### PR TITLE
Alma9 container build fixes rtj

### PR DESCRIPTION
**Pull request by rjones30**

### CODE REVIEW

**Comment by gluex** [comment]
Build status for this pull request: FAILURE
 
 Build log: [/work/halld/pull_request_test/halld_sim^alma9_container_build_fixes_rtj/make_alma9_container_build_fixes_rtj.log](https://halldweb.jlab.org/pull_request_test/halld_sim^alma9_container_build_fixes_rtj/make_alma9_container_build_fixes_rtj.log)
 Build report: [/work/halld/pull_request_test/halld_sim^alma9_container_build_fixes_rtj/report_alma9_container_build_fixes_rtj.txt](https://halldweb.jlab.org/pull_request_test/halld_sim^alma9_container_build_fixes_rtj/report_alma9_container_build_fixes_rtj.txt)
 Location of build: /work/halld/pull_request_test/halld_sim^alma9_container_build_fixes_rtj

File: src/SBMS/sbms.py [hunk 1]
```diff
@@ -1194,7 +1195,7 @@ def AddEvtGen(env):
 	else:
                 AddHepMC(env)
                 AddPhotos(env)
-                EVTGEN_CPPPATH = "%s/" % (EVTGEN_HOME)
+                EVTGEN_CPPPATH = "%s/include" % (EVTGEN_HOME)
```
Comment (aaust) [inline src/SBMS/sbms.py:1198]: This breaks the convention in the build_scripts environment and is responsible for the failed test

**Comment by rjones30** [comment]
Yes, it looks like you have the same problem as in DIRACXX, did you ever
complete the EVTGEN build with "make install"? That is what creates the
include directory and populates it.
-Richard Jones

**Comment by aaust** [comment]
It looks like we only did the usual "make -C", but halld_sim compiled fine with it. We can add a "make install" target to the Makefile if it is required.

**Comment by rjones30** [comment]
Alex, we should follow the standard CMAKE procedure: (cmake .. && cmake
--build && cmake install) for all of our cmake packages. This makes the
behavior uniform. Look at HEPMC and you will see the /include in the CPP
prefix defined in sbms.py, also see PHOTOS. It really helps to make this
stuff uniform across packages, and this is a good moment to make this
change, when we are stepping to a major new release.
-Richard Jones

**Comment by gluex** [comment]
Test status for this pull request: SUCCESS
 
  Summary: [/work/halld/pull_request_test/halld_sim^alma9_container_build_fixes_rtj/tests/summary.txt](https://halldweb.jlab.org/pull_request_test/halld_sim^alma9_container_build_fixes_rtj/tests-summary.txt)
 Logs: [/work/halld/pull_request_test/halld_sim^alma9_container_build_fixes_rtj/tests/log](https://halldweb.jlab.org/pull_request_test/halld_sim^alma9_container_build_fixes_rtj/tests-logs)
 
 Build log: [/work/halld/pull_request_test/halld_sim^alma9_container_build_fixes_rtj/make_alma9_container_build_fixes_rtj.log](https://halldweb.jlab.org/pull_request_test/halld_sim^alma9_container_build_fixes_rtj/make_alma9_container_build_fixes_rtj.log)
 Build report: [/work/halld/pull_request_test/halld_sim^alma9_container_build_fixes_rtj/report_alma9_container_build_fixes_rtj.txt](https://halldweb.jlab.org/pull_request_test/halld_sim^alma9_container_build_fixes_rtj/report_alma9_container_build_fixes_rtj.txt)
 Location of build: /work/halld/pull_request_test/halld_sim^alma9_container_build_fixes_rtj

**Review by aaust — APPROVED** [review]
Make install was added to the evtgen makefile in the build_script branch [separate_hddm](https://github.com/JeffersonLab/build_scripts/tree/separate_hddm) to compile this PR.

**Pull request merged**